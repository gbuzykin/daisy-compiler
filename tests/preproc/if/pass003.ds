#if true
#info "case #1"
#elif true
#info "case #2"
#endif

#if true
#info "case #3"
#elif false
#info "case #4"
#endif

#if false
#info "case #5"
#elif true
#info "case #6"
#endif

#if false
#info "case #7"
#elif false
#info "case #8"
#endif
