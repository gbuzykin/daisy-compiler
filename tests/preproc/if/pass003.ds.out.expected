./preproc/if/pass003.ds:2:2: info: "case #1"
 2 | #info "case #1"
   |  ^~~~
./preproc/if/pass003.ds:8:2: info: "case #3"
 8 | #info "case #3"
   |  ^~~~
./preproc/if/pass003.ds:16:2: info: "case #6"
 16 | #info "case #6"
    |  ^~~~
./preproc/if/pass003.ds: info: warnings 0, errors 0
